apps/web/src/pages/TermsAndConditions/TermsAndConditions.tsx

```diff
@@ -0,0 +1,856 @@
+                size="h3"
+                data-h2-margin="base(x3, 0, x1, 0)"
+              >
+                {sections.linkingToGov.title}
+              </TableOfContents.Heading>
+              <p data-h2-margin="base(x.5, 0)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Links to websites not under the control of the Government of Canada, including those to our social media accounts, are provided solely for the convenience of our website visitors. We are not responsible for the accuracy, currency or reliability of the content of such websites. The Government of Canada does not offer any guarantee in that regard and is not responsible for the information found through these links, and does not endorse the sites and their content.",
+                  id: "Jz7jAF",
+                  description: "Paragraph describing linking to gov section",
+                })}
+              </p>
+              <p data-h2-margin="base(x.5, 0)">
+                {intl.formatMessage(
+                  {
+                    defaultMessage:
+                      "Visitors should also be aware that information offered by non-Government of Canada sites to which this website links is not subject to the <privacyActLink>Privacy Act</privacyActLink> or the <langActLink>Official Languages Act</langActLink> and may not be accessible to persons with disabilities. The information offered may be available only in the language(s) used by the sites in question. With respect to privacy, visitors should research the privacy policies of these non-government websites before providing personal information.",
+                    id: "1e84jV",
+                    description:
+                      "Paragraph two describing linking to gov section",
+                  },
+                  {
+                    privacyActLink,
+                    langActLink,
+                  },
+                )}
+              </p>
+            </TableOfContents.Section>
+            <TableOfContents.Section id={sections.ownershipAndUsage.id}>
+              <TableOfContents.Heading
+                size="h3"
+                data-h2-margin="base(x3, 0, x1, 0)"
+              >
+                {sections.ownershipAndUsage.title}
+              </TableOfContents.Heading>
+              <p data-h2-margin="base(x.5, 0)">
+                {intl.formatMessage({
+                  id: "p8o9cO",
+                  defaultMessage:
+                    "Materials on this website were produced and/or compiled for the purpose of providing Canadians with access to information about the programs and services offered by the Government of Canada. You may use and reproduce the materials as follows:",
+                  description:
+                    "Paragraph describing ownership and usage section",
+                })}
+              </p>
+              <ul>
+                <li>
+                  <p data-h2-font-weight="base(bold)">
```

Comment:
We need to use `700` as the font-weight.

Reply:
bold is set to 700. Unless your asking to change it for another reason?
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/1924c7a4-45e8-4534-99d0-7d9720497948)

Reply:
Wait, seriously? I distinctly remember having a conversation in slack saying that this was not possible 😕

Reply:
@substrae did we get the `bold` key working at some point? 😅

Reply:
@yonikid15 @esizer 

`bold` has always worked, because it is 1 of 2 keywords that are natively built into CSS (the other being `normal`, which is equivalent to `400`). Keywords for any other weight don't work (e.g. extrabold, thin, light, etc.) because they aren't actually implemented in CSS. I do have a PR with custom configured keywords for font weights done already, I just haven't wrapped up the [other things in my list for the release](https://github.com/orgs/hydrogen-css/projects/3) yet. I'll look at pushing the release out sooner than later and moving the outstanding work to a subsequent one.